pages/createOrganization.js

```diff
@@ -25,7 +25,7 @@ class CreateOrganizationPage extends React.Component {
     }
 
     return (
-      <Page>
+      <Page title="Create Organization">
```

Comment:
Intl.

Reply:
Removed this one, thinking the nav title is not necessary on certain pages that have a big title on the page itself